cloud: add release notes for 20221227 (#11744)
pingcap/docs: tidb-cloud/release-notes-2022.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.

## December 27, 2022

**General changes**

> 

Author: ```suggestion

+ Upgrade the default TiDB version of new [Serverless Tier](https://docs.pingcap.com/tidbcloud/select-cluster-tier#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).

```

Author: @baiyuqing PTAL

Reviewer: ```suggestion

+ Upgrade the default TiDB version of new [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).

```
Rest LGTM

- The point-in-time recovery (PITR) for Dedicated Tier clusters is now in General Availability (GA).

    PITR supports restoring data of any point in time to a new cluster. To use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0 and the TiKV node size is at least 8 vCPU and 16 GiB.

>     You can turn on or off the PITR feature in the Backup Settings of the [TiDB Cloud console](https://tidbcloud.com).

Author: ```suggestion
    You can turn on or off the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```

>     You can turn on or off the PITR feature in the Backup Settings of the [TiDB Cloud console](https://tidbcloud.com).

Reviewer 2: ```suggestion
    You can turn on or off the PITR feature in **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```

>     You can turn on or off the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).

Reviewer 3: ```suggestion
    You can enable or disable the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```

    For more information, see [Back up and restore TiDB cluster data](/tidb-cloud/backup-and-restore.md).

> - Support managing multiple changefeeds and editing existing changefeeds in the TiDB Cloud console.

Reviewer: ```suggestion
- Support managing multiple changefeeds and editing existing changefeeds.
```

>     - You can now create as many changefeeds as needed to manage different replicated data. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md)

Reviewer 2: ```suggestion
    - You can now create as many changefeeds as needed to manage different replicated data. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md).
```

>     - You can now create as many changefeeds as needed to manage different replicated data. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md).

Reviewer 3: ```suggestion
    - You can now create as many changefeeds as needed to manage different data replication tasks. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md).
```

>     - You can edit the configuration of an existing changefeed in the paused status. For more details, refer to [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed)

Reviewer 2: ```suggestion
    - You can edit the configuration of an existing changefeed in the paused status. For more details, refer to [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed).
```

>     - You can edit the configuration of an existing changefeed in the paused status. For more details, refer to [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed).

Reviewer 3: ```suggestion
    - You can edit the configuration of an existing changefeed in the paused status. For more information, see [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed).
```

- Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-hosted MySQL-compatible databases to TiDB Cloud online. This feature is now in General Availability.

    - Provide services in the following 6 regions:
        - AWS Oregon (us-west-2)
        - AWS N. Virginia (us-east-1)
        - AWS Mumbai (ap-south-1)
        - AWS Singapore (ap-southeast-1)
        - AWS Tokyo (ap-northeast-1)
        - AWS Frankfurt (eu-central-1)
    - Support multiple specifications. You can choose an appropriate specification according to the required performance to achieve optimal data migration experience.

>     For how to migrate data to TiDB Cloud, refer to [user documentation](/tidb-cloud/migrate-from-mysql-using-data-migration.md). For billing details, refer to [Data Migration Billing](/tidb-cloud/tidb-cloud-billing-dm.md).

Reviewer: ```suggestion
  For how to migrate data to TiDB Cloud, refer to [user documentation](/tidb-cloud/migrate-from-mysql-using-data-migration.md). For billing details, refer to [Data Migration billing](/tidb-cloud/tidb-cloud-billing-dm.md).
```

- Support importing local CSV files to TiDB Cloud.

>     It only takes a few clicks to complete the task configuration, and then your local CSV data will be quickly imported into your TiDB cluster. When using this method, you do not need to provide the cloud storage bucket path and Role ARN. The whole importing process is quick and smooth.

Reviewer 3: ```suggestion
    It only takes a few clicks to complete the task configuration, and then your local CSV data can be quickly imported into your TiDB cluster. When using this method, you do not need to provide the cloud storage bucket path and Role ARN. The whole importing process is quick and smooth.
```

>     For more information, refer to [Import Local Files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).

Reviewer 2: ```suggestion
    For more information, refer to [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

>     For more information, refer to [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).

Reviewer 3: ```suggestion
    For more information, see [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

>     For more information, refer to [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).

Reviewer 3: ```suggestion
    For more information, see [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

>     For more information, refer to [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).

Reviewer 3: ```suggestion
    For more information, see [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.

## December 27, 2022

**General changes**

> + Upgrade the default TiDB version of all [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).

Author: ```suggestion
- Upgrade the default TiDB version of all [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
```

> - Upgrade the default TiDB version of all [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).

Author: ```suggestion
- Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
```
